src/pkg/packager/load/load.go

```diff
@@ -31,6 +31,8 @@ type DefinitionOptions struct {
 	CachePath string
 	// IsInteractive decides if Zarf can interactively prompt users through the CLI
 	IsInteractive bool
+	// BypassVersionCheck skips version requirement validation
+	BypassVersionCheck bool
```

Comment:
Thoughts on using "SkipVersionCheck" as the name for this field in various places? It offers some consistency with existing language.